src/Sentry.Unity.iOS/SentryNativeCocoa.cs

```diff
@@ -14,15 +15,14 @@ public static class SentryNativeCocoa
     /// Configures the native support.
     /// </summary>
     /// <param name="options">The Sentry Unity options to use.</param>
-    /// <param name="sentryUnityInfo">Infos about the current Unity environment</param>
-    public static void Configure(SentryUnityOptions options, ISentryUnityInfo sentryUnityInfo) =>
-        Configure(options, sentryUnityInfo, ApplicationAdapter.Instance.Platform);
+    public static void Configure(SentryUnityOptions options) =>
+        Configure(options, SentryPlatformServices.UnityInfo, ApplicationAdapter.Instance.Platform);
 
-    internal static void Configure(SentryUnityOptions options, ISentryUnityInfo sentryUnityInfo, RuntimePlatform platform)
+    internal static void Configure(SentryUnityOptions options, ISentryUnityInfo? sentryUnityInfo, RuntimePlatform platform)
     {
         options.DiagnosticLogger?.LogInfo("Attempting to configure native support via the Cocoa SDK");
 
-        if (!sentryUnityInfo.IsNativeSupportEnabled(options, platform))
+        if (!sentryUnityInfo?.IsNativeSupportEnabled(options, platform) ?? false)
```

Comment:
same situation where it's not clear if it's a mix up with setting up this static thing before calling this, or `IsNativeSupportEnabled` returning `false`, and making things hard to debug